pysatNASA/constellations/icon.py

```diff
@@ -1,4 +1,11 @@
-"""Creates a constellation from NASA the ICON satellite platform."""
+"""Creates a constellation from NASA the ICON satellite platform.
+
+Includes the core instruments without the line of sight winds.
+
+Note that IVM A and B are nominally never active at the same time. This
+constellation requires a bug-fix in pysat 3.1.0.
```

Comment:
So, question:  Does it require a bug fix, or does it require setting the `common_index=False` until there's a bug fix? I tried testing it locally, but ran into a conflict with netCDF and numpy that needs to be sorted 🐞

Reply:
Hmm.  I just tried this for the icon constellation and loading worked fine.  Are you testing against pysat main or develop?

I think that the common_index setting is the correct solution.  Need to update my pysat pull accordingly.

Reply:
It's a netCDF version issue.  I ended up solving it later today, because it kept being annoying.  Just took some time to do all the software updates and I figured the `common_index` solution would work.